docs/data-prep/ingest-data-fs.md

```diff
@@ -198,12 +198,13 @@ Tech Preview
 ```
 
 The Redis online target is called, in MLRun, `RedisNoSqlTarget`. The functionality of the `RedisNoSqlTarget` is identical to the `NoSqlTarget` except for:
-- The `RedisNoSqlTarget` accepts the path parameter in the form `<redis|rediss>://[<username>]:[<password>]@<host>[:port]`<br>
-For example: `rediss://:abcde@localhost:6379` creates a redis target, where:
+- The RedisNoSqlTarget accepts the path parameter in the form: `<redis|rediss>://<host>[:port]`
+For example: `rediss://localhost:6379` creates a redis target, where:
    - The client/server protocol (rediss) is TLS protected (vs. "redis" if no TLS is established)
-   - The server is password protected (password="abcde")
    - The server location is localhost port 6379.
-- A default path can be configured in redis.url config (mlrun client has priority over mlrun server), and can be overwritten by `MLRUN_REDIS__URL` env var.
+- If the path parameter is not set, it tries to fetch it from the MLRUN_REDIS__URL environment variable.
+- You cannot pass the username/password as part of the URL. If you want to provide the username/password, use secrets as:
+`<prefix_>REDIS_USER <prefix_>REDIS_PASSWORD` where <prefix> is the optional RedisNoSqlTarget `credentials_prefix` parameter.
```

Comment:
Note that K8s secrets are not available when executing locally (from the sdk).
Therefore if RedisNoSqlTarget with secret is used, one has to add the secret as env-var